### ISSUE TITLE
Add Dependency-Management Dependencies

### ISSUE THREAD

**Issue by MBoegers** [openrewrite/rewrite-recipe-bom#34]
## What problem are you trying to solve?
The Operaton Project forked Camunda 7 and plans to provide an easy migration for potential users, their efforts are open via [operaton/migrate-from-camunda-recipe](https://github.com/operaton/migrate-from-camunda-recipe).
One problem is to migrate the dependencies from the Camunda namespace to the Operaton namespace.
While writing test, it occurred that the dependency managent recipes and tooling are not fully managed by the _org.openrewrite.recipe:rewrite-recipe-bom_ artifact.

Needed additions to develop/test Maven and Gradle migrations:

- org.openrewrite.gradle.tooling:model
- org.codehaus.groovy:groovy, scope test
- org.gradle:gradle-tooling-api, scope test

## Describe the solution you'd like
Manage the needed dependencies for framework developer to lower the barrier.

## Have you considered any alternatives or workarounds?
Manual adding and writing a specially shaped documentation could be a workaround.

**Comment by timtebeek** [comment]
Yes makes sense for those not using our build plugins I suppose; feel free to add these and push them up!

**Comment by timtebeek** [comment]
The place to add these is likely near these lines, as those are similarly not recipe libraries, but co-managed versions.
https://github.com/openrewrite/rewrite-recipe-bom/blob/e16ceba99e9588bf74669ac6ea036d6191c8f41d/build.gradle.kts#L25-L28

**Comment by timtebeek** [comment]
rewrite-bom already manages org.openrewrite.gradle.tooling:model (and plugin), pinned in lockstep with rewrite core (e.g. 8.84.9).

In terms of Groovy we support a wide range of versions, and don't really want to prescribe or manage third party dependencies through our BOMs, as that creates additional maintenance work. I think we can safely close this issue now.
